www/src/components/__tests__/banner.js

```diff
@@ -6,7 +6,7 @@ import Banner from "../banner"
 test(`it renders an external link`, () => {
   const { getByText } = render(<Banner />)
 
-  const link = getByText(`Register now`)
+  const link = getByText(`Watch now`)
```

Comment:
Heh - we should fix this. Don't want to tweak a test every time we want to change the text.

Reply:
👍